prereise/gather/griddata/hifld/data_access/load.py

```diff
@@ -0,0 +1,10 @@
+import pandas as pd
+
+
+def load_csv(csv_filename, dtypes=None):
+    if dtypes:
+        data = pd.read_csv(csv_filename, dtype=dtypes)
+    else:
+        data = pd.read_csv(csv_filename)
+
+    return data
```

Comment:
Is the `dtypes` used somewhere? Just asking because it is not used in the `get_grid_data` function of the `model` module

Reply:
Oops, it is. Sorry about that.